packages/web/src/pages/upload-page/fields/SourceFilesView.tsx

```diff
@@ -165,7 +166,9 @@ const StemListItem = ({
           textClassName={styles.dropdownText}
         />
       </div>
-      <div className={styles.title}>{metadata.title}</div>
+      <Text size='Small' strength='Strong'>
```

Comment:
do these need to be capitalized? idk why but I assumed we'd use lower

Reply:
yea Randy, brought this up too. I have an update to make the all lower coming next 👍